### ISSUE TITLE
Suggestion: Exported telemetry

### ISSUE THREAD

**Issue by AndrewDTR** [Ninjabrain1/Ninjabrain-Bot#76]
Hey! Just thought it would be cool if the bot had a feature where, along with the OBS image it dumps, it could output a basic .txt file with the data it displays on screen. This could be as simple as just displaying the percentage and the distance, though I'm sure if you wanted you could put all of the data. This would let people customize/drive displays other than the OBS display. Thanks!

**Comment by krankyy** [comment]
I thinks its already done in some way cuz i saw this in other  peoples recordings

**Comment by Ninjabrain1** [comment]
Maybe, I dont know how large the demand is though. 

Regarding what krankyy said, it is no imlemented yet. Maybe you are thinking of people who remove the background with OBS, like a greenscreen.

**Comment by AndrewDTR** [comment]
> Maybe, I dont know how large the demand is though.
> 
> Regarding what krankyy said, it is no imlemented yet. Maybe you are thinking of people who remove the background with OBS, like a greenscreen.

Yeah, that's how I was inspired to suggest this. People have made with different ways of overlaying the provided image, but I just generally think that the community would have more interesting ways of playing around with a text file. A little while ago I made a rudimentary fork of this suggestion (with the bot just dumping the probabilities & distances to a text file on my desktop) and it let me make some cool visualizations -- I've attached it to this comment if you want to take a look. I don't think it would be too hard to implement into the calculator, and I can't envision a downside or any risk (as the bot already gives an image, and people could just OCR the known positions of the information in those images to create the same thing -- this recommendation exists as a way to avoid having to do that). Thanks!


https://github.com/Ninjabrain1/Ninjabrain-Bot/assets/74398819/191a8359-f288-4ff6-9fe3-0a1c02ce4b9a
